11-node-frameworks/classworks/Malkov_Aleksei/session-counter.js

```diff
@@ -0,0 +1,30 @@
+let session = require('koa-generic-session');
```

Comment:
Let's compose this and main server realisation into one server. Keep in mind that the session counter should work for every request without adding additional code if you add a new endpoint. TIP: check how body-parser works

Reply:
i made some changes, maybe i didnt get it, but now it should count on every request on main server